idaes/power_generation/carbon_capture/compression_system/compressor.py

```diff
@@ -522,8 +522,8 @@ def psi_s_stage_eqn(b, t):
                 return b.psi_s[t] == b.coeff_a * b.psi_3[t]**2 + \
                     b.coeff_b * b.psi_3[t] + b.coeff_c
 
-    def initialize(self, state_args={}, outlvl=idaeslog.NOTSET, solver='ipopt',
-                   optarg={'tol': 1e-6, 'max_iter': 500}):
+    def initialize(blk, state_args={}, outlvl=idaeslog.NOTSET, solver='ipopt',
+                   optarg={'tol': 1e-6, 'max_iter': 200}):
```

Comment:
A max iter of 200 is huge for an initialization routine. Normally we do not even set this, but it you do it probably should be only 50 (or less) unless there is a good reason for it to be larger.

Reply:
Here I think self is more the python standard, so its probably best to stick to self.  The main reason you'll see code with blk rather than self is that a long time ago we we're using a different style less object-oriented style where we had functions that just took Pyomo block as an argument and did something to it.